domains/whattyu.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "My Website",
+  "repo": "https://github.com/whattyu/website",
+  "owner": {
+    "username": "whattyu",
+    "email": "dulunatatolga@gmail.com"
+  },
+  "record": {
+    "CNAME": "whattyu.vercel.app"
```

Comment:
vercel cant be used sowwy

instead try [railway](https://railway.app)

Reply:
railway gives me an error but in vercel isn't

Reply:
github pages too

Reply:
idc

Reply:
our service doesn't work with vercel/netlify, sowwy :(